demo/build.gradle

```diff
@@ -19,7 +19,7 @@ android {
         versionCode 1
         versionName "1.0"
 
-        manifestPlaceholders = [downloadAuthority: "${applicationId}"]
+        manifestPlaceholders = [downloadAuthority: "${applicationId}.downloads"]
```

Comment:
We want the client to either completely control authority creation, or be completely agnostic of it (and we just use context). Not in the middle